tests/integration/conftest.py

```diff
@@ -230,9 +226,74 @@ def member(pg_driver):
 
 
 @pytest.fixture()
-def cgci_blgsp(client, admin):
-    put_program(client, admin)
-    put_cgci_blgsp(client, admin)
+def put_program(client, admin):
+    def put_program_helper(headers=admin, name='CGCI', phsid='phs000235', status_code=200):
```

Comment:
I'm pretty sure these default args are going to be sticky, which shouldn't be a problem because this is functionally scoped, but would be a problem if it were not functionally scoped.